### DISCUSSION TITLE
AlDente 1.10

### DISCUSSION THREAD

**Discussion by davidwernhart** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#457] [General]
* Fixed an issue that could cause charging above the set limit when heat protection mode was enabled
* Improved the set-up experience
* Added Spanish, Italian, French and Russian language support. Thank you very much to all community members who contributed to make this happen! :)
* AlDente now shows the UI immediately upon start
* Fixed a bug that caused strange behavior when using AlDente on multiple user accounts. When logged off, AlDente now behaves the same as if it would be closed
* Completely revamped the sleep system. This fixes a lot of bugs related to Discharge, Calibration Mode and Clamshell Mode. There is now also a toggle that lets you disable sleep completely as long as the charge limit is not yet reached. This means you can set the limit, close your laptop and it will only start to go to sleep when the desired limit is reached. You can read more about this feature here: [https://apphousekitchen.com/feature-explanation-disable-sleep-until-charge-limit/](https://apphousekitchen.com/feature-explanation-disable-sleep-until-charge-limit/)

Enjoy your Pasta,
David & Matthias

<hr /><em>This discussion was created from the release <a href='https://github.com/davidwernhart/AlDente/releases/tag/1.10'>AlDente 1.10</a>.</em>

**Comment by charli0209** [comment]
why i cannot download it

**Reply by MatthiasKerbl** [reply]
Hi @charli0209,

You can download it here: https://github.com/davidwernhart/AlDente/releases/download/1.10/AlDente1.1.dmg

Cheers,
Matthias

**Comment by charli0209** [comment]
Hi Matthias,

Thanks for your reply!  I have started to use it! Have a nice day!

Cheers,
Charli

发件人: Matthias Kerbl ***@***.***>
日期: 星期二, 2021年11月23日 11:11
收件人: davidwernhart/AlDente ***@***.***>
抄送: charli0209 ***@***.***>, Mention ***@***.***>
主题: Re: [davidwernhart/AlDente] AlDente 1.10 (Discussion #457)

Hi @charli0209<https://github.com/charli0209>,

You can download it here: https://github.com/davidwernhart/AlDente/releases/download/1.10/AlDente1.1.dmg

Cheers,
Matthias

―
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/davidwernhart/AlDente/discussions/457#discussioncomment-1686861>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AWUBR4RJZKSTQOP2ICDBZ2LUNNZFBANCNFSM5IBFIJHQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
